.cirrus.yml

```diff
@@ -360,30 +342,32 @@ meta_task:
     script: '$CIRRUS_WORKING_DIR/$SCRIPT_BASE/update_meta.sh |& ${TIMESTAMP}'
 
 
+# Temporarily disabled in anticipation of moving this workflow into
+# containers/automation_images
 # Remove old and disused images based on labels set by meta_task
-image_prune_task:
-
-    # This should ONLY ever run from the master branch, and never
-    # anywhere else so it's behavior is always consistent, even
-    # as new branches are created.
-    only_if: $CIRRUS_BRANCH == "master"
-
-    depends_on:
-        - "meta"
-
-    container:
-        image: "quay.io/libpod/imgprune:master"  # see contrib/imgprune
-        cpu: 1
-        memory: 1
-
-    env:
-        <<: *meta_env_vars
-        GCPJSON: ENCRYPTED[766916fedf780cbc16ac3152f7f73c5d9dcf64768fc6e80b0858c5badd31e7b41f3c864405c814189fd340e5a056ba18]
-        GCPNAME: ENCRYPTED[d6869741209b8cf380adb8a3858cbce4542c9cf115452fcd2024a176b08fce10112e8bf0fbcc2f0033e7b87ef4342b3a]
-
-    timeout_in: 10m
-
-    script: '/usr/local/bin/entrypoint.sh |& ${TIMESTAMP}'
+# image_prune_task:
```

Comment:
Is there a reason this is just commented out and not deleted?

Reply:
I'm not 100% sure if I want to re-enable it temporarily while moving it over to `automation_images` repo.  It's quite a bit of work to move, so depending on how long that takes, it might need to be switched back on temporarily to clean out a "too large" set of stale/orphan/testing VM images.  I'm not dead-set against simply removing it, if you think it's distracting.